package.json

```diff
@@ -255,37 +255,49 @@
       "default": "automatic",
       "order": 17
     },
+    "openerSplit": {
```

Comment:
Ok, how about `pdfViewSplitDirection` and we can change it to `openerSplitDirection` if we actually add other viewers and we want to have a single unified setting?